source/config/mockData/index.js

```diff
@@ -0,0 +1,20 @@
+//Packages
+import { createServer } from "miragejs"
```

Comment:
What advantages does this library has?
I mean, we use Axios for make HTTP calls, and axios-mock-adapter for creating the mock services, why should we use this miragejs instead?